### PR TITLE
[IOCOM-1502] Setup msgs-sending-func

### PR DESCRIPTION
**Pull request by michaeldisaro**
This branch is going to setup `msgs-sending-func` app from `io-functions-services-messages` repo.
This setup involves:
* updating node to v20
* updating some dependencies
* fixing tests to run with new environment
* added github actions
* finalized infra for deployment (added redis and props)

### CODE REVIEW

File: apps/msgs-sending-func/.yarn/install-state.gz
Comment (Garma00) [inline apps/msgs-sending-func/.yarn/install-state.gz:1]: I don't think we want to commit this file https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored

File: apps/msgs-sending-func/__integrations__/.yarn/install-state.gz
Comment (Garma00) [inline apps/msgs-sending-func/__integrations__/.yarn/install-state.gz:1]: Ditto

**Review by Garma00 — APPROVED** [review]
LGTM

File: apps/msgs-sending-func/.yarn/install-state.gz
Comment (michaeldisaro) [reply apps/msgs-sending-func/.yarn/install-state.gz:1]: @lucacavallaro

**Review by acuiuli — APPROVED** [review]
LGTM

**Pull request merged**